src/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater/ConfigContext.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater
+{
+    public class ConfigContext
+    {
+        public Dictionary<string, ServiceSpecificContext> ServiceContext { get; }
+
+        public bool NetTcpCertificate { get; }
+
+        public bool HasMetadata { get; }
+
+        public bool WindowsAuthentication { get; }
+
+        public class ServiceSpecificContext
+        {
+            public Dictionary<string, Uri> SchemeToAddressMapping { get; }
+
+            public MetadataType MetadataType { get; }
+
+            public Dictionary<string, string> ServiceDebug { get; }
+
+            public HashSet<string> Bindings { get; }
+
```

Comment:
Is this the set of binding types, or binding configuration names? Can you make the name a little more description?

Reply:
This is the set of binding types, and yes I will improve the name!